resend.yaml

```diff
@@ -1404,3 +1581,270 @@ components:
+          description: The plain text version of the template.
+        variables:
+          type: array
+          items:
+            $ref: '#/components/schemas/TemplateVariable'
+        created_at:
+          type: string
+          format: date-time
+          description: Timestamp indicating when the template was created.
+        updated_at:
+          type: string
+          format: date-time
+          description: Timestamp indicating when the template was last updated.
+        status:
+          type: string
+          description: The publication status of the template.
+          enum: [draft, published]
+        published_at:
+          type: string
+          format: date-time
+          description: Timestamp indicating when the template was published.
+          nullable: true
+        has_unpublished_versions:
+          type: boolean
+          description: Indicates whether the template has unpublished versions.
+    TemplateListItem:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The ID of the template.
+        name:
+          type: string
+          description: The name of the template.
+        status:
+          type: string
+          description: The publication status of the template.
+          enum: [draft, published]
+        published_at:
+          type: string
+          format: date-time
+          nullable: true
+          description: Timestamp indicating when the template was published.
+        created_at:
+          type: string
+          format: date-time
+          description: Timestamp indicating when the template was created.
+        updated_at:
+          type: string
+          format: date-time
+          description: Timestamp indicating when the template was last updated.
+        alias:
+          type: string
+          description: The alias of the template.
+    CreateTemplateRequest:
+      type: object
+      required:
+        - name
+        - html
+      properties:
+        name:
+          type: string
+          description: The name of the template.
+        alias:
+          type: string
+          description: The alias of the template.
+        from:
+          type: string
+          description: Sender email address. To include a friendly name, use the format "Your Name <sender@domain.com>".
+        subject:
+          type: string
+          description: Email subject.
+        reply_to:
+          type: array
+          items:
+            type: string
+          description: Reply-to email addresses.
+        html:
+          type: string
+          description: The HTML version of the template.
+        text:
+          type: string
+          description: The plain text version of the template.
+        variables:
+          type: array
+          items:
+            $ref: '#/components/schemas/TemplateVariableInput'
+    CreateTemplateResponseSuccess:
+      type: object
+      properties:
```

Comment:
According to the docs it seems like we're missing this the `object` property with the type indication

https://resend.com/docs/api-reference/templates/create-template